### PR TITLE
added test with invalid table indices

### PR DESCRIPTION
**Pull request by wanderer**
related https://github.com/WebAssembly/spec/issues/404

### CODE REVIEW

File: interpreter/test/call_indirect.wast [hunk 1]
```diff
@@ -360,3 +360,12 @@
   )
   "unknown type"
 )
+
+;; invalid table
+(assert_invalid 
+  (module
+    (table anyfunc (elem 0 $t1 44))
+    (func $t1 (result i32) (i32.const 1))
```
Comment (rossberg-chromium) [inline interpreter/test/call_indirect.wast]: What is the purpose of including $t1 in this test?

Comment (wanderer) [inline interpreter/test/call_indirect.wast]: I did have mutlpe functions there, I have updated to the minimal test that wabt was failing at.

Comment (rossberg-chromium) [inline interpreter/test/call_indirect.wast]: Ah, I actually meant the entire function, not just the name. AFAICS, the minimal test would be just
```
(module (table anyfunc (elem 0 0)))
```

Comment (wanderer) [inline interpreter/test/call_indirect.wast]: ah good point! Updated

**Pull request merged**